sycl/test-e2e/Matrix/XMX8/joint_matrix_transposeC.cpp

```diff
@@ -6,12 +6,11 @@
 //
 //===----------------------------------------------------------------------===//
 // REQUIRES: matrix-xmx8
+// REQUIRES-INTEL-DRIVER: lin: 28267
```

Comment:
do we know driver version for transpose now?
If yes please add it to the PVC test.

Reply:
I don't want to add it to the PVC testing, as it will stop running in my local CI, since I'm using the latest Agama and the fix is not in Agama yet.